README.md

```diff
@@ -1,106 +1,123 @@
-
-Features:
--------
+### :white_check_mark: Implemented
 
 ```
-STRAIGHT UP
-
 As a Maker
 So that I can let people know what I am doing  
 I want to post a message (peep) to chitter
+```
 
+```
 As a maker
 So that I can see what others are saying  
 I want to see all peeps in reverse chronological order
+```
 
+```
 As a Maker
 So that I can better appreciate the context of a peep
 I want to see the time at which it was made
+```
 
+```
 As a Maker
 So that I can post messages on Chitter as me
 I want to sign up for Chitter
+```
 
-HARDER
-
+```
 As a Maker
 So that only I can post messages on Chitter as me
 I want to log in to Chitter
+```
 
+```
 As a Maker
 So that I can avoid others posting messages on Chitter as me
 I want to log out of Chitter
+```
 
-ADVANCED
+### :warning: To implement
 
+```
 As a Maker
 So that I can stay constantly tapped in to the shouty box of Chitter
 I want to receive an email if I am tagged in a Peep
 ```
 
-Technical Approach:
------
 
-In this unit, you integrated a database into Bookmark Manager using the `PG` gem and `SQL` queries. You can continue to use this approach when building Chitter Challenge.
+## Notes on functionality:
 
-If you'd like more technical challenge now, try using an [Object Relational Mapper](https://en.wikipedia.org/wiki/Object-relational_mapping) as the database interface.
+:white_check_mark: You don't have to be logged in to see the peeps.
 
-Some useful resources:
-**DataMapper**
-- [Datamapper wiki](https://en.wikipedia.org/wiki/DataMapper)
-- [Sinatra, PostgreSQL & DataMapper recipe](https://github.com/sinatra/sinatra-recipes/blob/master/databases/postgresql-datamapper.md)
+:white_check_mark: Makers sign up to chitter with their username, name, lastname, email, password (e.g. sjmog, Sam, Morgan, samm@makersacademy.com, password123).
 
-**Ruby Object Mapper**
-- [ROM](https://rom-rb.org/)
+:white_check_mark: The username and email are unique.
 
-**ActiveRecord**
-- [ActiveRecord ORM](https://guides.rubyonrails.org/active_record_basics.html)
-- [Sinatra, PostgreSQL & ActiveRecord recipe](http://recipes.sinatrarb.com/p/databases/postgresql-activerecord?#article)
+:white_check_mark: Peeps (posts to chitter) have the name of the username.
 
-Notes on functionality:
-------
+:white_check_mark: Your README should indicate the technologies used, and give instructions on how to install and run the tests.
 
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
+## Languages used
 
-Bonus:
------
+1. Ruby
+2. RSpec
+3. HTML
+4. CSS
+5. Bootstrap
```

Comment:
Your README is very detailed and I love the fact that you have mentioned the languages used